json-pointer/json-pointer.iml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we keep IDE files outside the repo right? or should I remove it from .gitignore?